ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml

```diff
@@ -0,0 +1,355 @@
+        elif [[ "${CLUSTER_TYPE}" == "openstack" ]]; then
+          mkdir -p ~/.ssh
+          cp /tmp/cluster/ssh-privatekey ~/.ssh/kube_openstack_rsa || true
+        fi
+        mkdir -p /tmp/output
+        cd /tmp/output
+        # install telemetry client (TEMPORARY)
+        set +e
+        curl https://raw.githubusercontent.com/openshift/telemeter/master/deploy/default-rules > /tmp/rules
+        oc create ns openshift-telemetry
+        oc adm policy add-cluster-role-to-user cluster-reader -z default -n openshift-telemetry
+        oc create secret generic telemeter-client --from-literal=to=https://telemeter.svc.ci.openshift.org --from-literal=id=${NAMESPACE}-${JOB_NAME_HASH} --from-file=token=/tmp/cluster/telemeter-token --from-literal=salt=random-salt --from-file=rules=/tmp/rules --dry-run -o yaml | oc apply -n openshift-telemetry -f -
+        oc create -f https://raw.githubusercontent.com/openshift/telemeter/master/deploy/telemeter-client-openshift.yaml -n openshift-telemetry
+        set -e
+
+        ${TEST_COMMAND}
+
+    # Runs an install
+    - name: setup
+      image: ${IMAGE_INSTALLER}
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp
+      - name: cluster-profile
+        mountPath: /etc/openshift-installer
+      - name: artifacts
+        mountPath: /tmp/artifacts
+      env:
+      - name: TYPE
+        value: ${CLUSTER_TYPE}
+      - name: AWS_SHARED_CREDENTIALS_FILE
+        value: /etc/openshift-installer/.awscred
+      - name: OPENSHIFT_INSTALL_CLUSTER_NAME
+        value: ${NAMESPACE}-${JOB_NAME_HASH}
+      - name: OPENSHIFT_INSTALL_BASE_DOMAIN
+        value: origin-ci-int-aws.dev.rhcloud.com
+      - name: OPENSHIFT_INSTALL_EMAIL_ADDRESS
+        value: test@ci.openshift.io
+      - name: OPENSHIFT_INSTALL_PASSWORD
+        value: verysecure
+      - name: OPENSHIFT_INSTALL_SSH_PUB_KEY_PATH
+        value: /etc/openshift-installer/ssh-publickey
+      - name: OPENSHIFT_INSTALL_PULL_SECRET_PATH
+        value: /etc/openshift-installer/pull-secret
+      - name: OPENSHIFT_INSTALL_PLATFORM
+        value: ${CLUSTER_TYPE}
+      - name: OPENSHIFT_INSTALL_AWS_REGION
+        value: us-east-1
+      - name: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
+        value: ${RELEASE_IMAGE_LATEST}
+      - name: OPENSHIFT_INSTALL_OPENSTACK_IMAGE
+        value: rhcos
+      - name: OPENSHIFT_INSTALL_OPENSTACK_CLOUD
+        value: openstack-cloud
+      - name: OPENSHIFT_INSTALL_OPENSTACK_REGION
+        value: RegionOne
+      - name: OPENSHIFT_INSTALL_OPENSTACK_EXTERNAL_NETWORK
+        value: public
+      - name: OS_CLOUD
+        value: openstack-cloud
+      - name: OS_CLIENT_CONFIG_FILE
+        value: /etc/openshift-installer/clouds.yaml
+      - name: USER
+        value: test
+      - name: HOME
+        value: /tmp
+      command:
+      - /bin/sh
+      - -c
+      - |
+        #!/bin/sh
+        trap 'rc=$?; if test "${rc}" -ne 0; then touch /tmp/exit; fi; exit "${rc}"' EXIT
```

Comment:
```diff
-        trap 'rc=$?; if test "${rc}" -eq 0; then touch /tmp/setup-success; else touch /tmp/exit; fi; exit "${rc}"' EXIT
+        trap 'rc=$?; if test "${rc}" -ne 0; then touch /tmp/exit; fi; exit "${rc}"' EXIT
```

This one's still outstanding from #2062.

Reply:
To clarify - I *should* be checking `/tmp/setup-success`, right?